presto-main/src/main/java/io/prestosql/operator/OperatorContext.java

```diff
@@ -670,6 +675,7 @@ public void close()
                 throw new UnsupportedOperationException("Called close on unclosable local memory context");
             }
             delegate.close();
+            allocationListener.run();
```

Comment:
this isn't an allocation

Reply:
I believe allocation listener is triggered on allocation and deallocation.  In this case I think `delegate.close()` can free memory, so we would need to fire the listener.